sdk/storage/tests.yml

```diff
@@ -5,29 +5,14 @@ jobs:
     parameters:
       ServiceDirectory: storage
       TimeoutInMinutes: 120
+      Matrix:
```

Comment:
This will be removed, was added to limit testing test runs to only use a single VM.